Defs/ThingDefs_Misc/Weapons_Turrets.xml

```diff
@@ -115,6 +121,60 @@
         <reloadTime>7.8</reloadTime>
         <ammoSet>AmmoSet_145x114mm</ammoSet>
       </li>
+      <li Class="CombatExtended.CompProperties_FireModes">
+        <aimedBurstShotCount>5</aimedBurstShotCount>
```

Comment:
Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.